css-values-5/Overview.bs

```diff
@@ -111,6 +111,80 @@ Request URL Modifiers</h4>
 		given |req|.
 	</div>
 
+<h2 id="other-units">
+Other Quantities</h2>
+
+	See [[css-values-4#other-units]].
+
+<!--
+████████ ████████     ███      ██████  ████████ ████  ███████  ██    ██
+██       ██     ██   ██ ██    ██    ██    ██     ██  ██     ██ ███   ██
+██       ██     ██  ██   ██   ██          ██     ██  ██     ██ ████  ██
+██████   ████████  ██     ██  ██          ██     ██  ██     ██ ██ ██ ██
+██       ██   ██   █████████  ██          ██     ██  ██     ██ ██  ████
+██       ██    ██  ██     ██  ██    ██    ██     ██  ██     ██ ██   ███
+██       ██     ██ ██     ██   ██████     ██    ████  ███████  ██    ██
+-->
+
+<h3 id="fractions">
+Flexible lengths: the <<fraction>> type</h3>
+
+	Fraction values are denoted by <dfn id="flex-value">&lt;fraction></dfn>,
+	and indicate a value that is some fraction of another reference value.
+	A fraction is a <<dimension>>.
+	Its unit identifier is <dfn value for="<flex>" lt="fr|fr unit">fr</dfn>.
+
+	These values specify
+	<dfn export lt="flexible length">flexible lengths</dfn>.
+	Each length defines a portion of a given space
+	which is specified by the context the type is used in.
+
+	The allowed range of <<fraction>>> values
+	<em>always</em> excludes negative values,
+	in addition to any explicit ranges that might be specified.
```

Comment:
Let's actually define what this thing does...

```suggestion
Proportional Fractions: the <<fraction>> type</h3>

	<dfn id=flex-fraction>Proportional fractions</dfn> are denoted by <dfn id="flex-value">&lt;fraction></dfn>,
	which is <<dimension>> with the <dfn value for="<flex>" lt="fr|fr unit">fr</dfn>unit.
	They represent a fraction of a reference value
	(typically a length)
	as a proportion in relation to other [=flex fractions=] referencing that same value,
	i.e. a value of ''<var>N</var>fr'' represents <var>N</var> / max(<var>sum of all ''fr'' values</var>, 1).
	For example, leftover space in [=grid layout=]
	is distributed to a ''1fr'' track and a ''2fr'' track in a 1:2 ratio.

	The allowed range of <<fraction>>> values
	<em>always</em> excludes negative values,
	in addition to any explicit ranges that might be specified.
```

Reply:
Thanks! Reads better now.